docs/intro/getting-started-with-mkdocs.rst

```diff
@@ -62,6 +62,11 @@ you can start using Read the Docs by :doc:`importing your docs </intro/import-gu
 
 .. _install MkDocs: https://www.mkdocs.org/#installation
 
+.. warning::
```

Comment:
This banner is good but it's a little close to the image which causes some awkwardness.

![Screen Shot 2020-05-08 at 2 43 15 PM](https://user-images.githubusercontent.com/185043/81451847-4598e480-913a-11ea-8c3e-be3b65cd3a70.png)